w2w/w2w.py

```diff
@@ -69,7 +83,18 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
         dest='wrf_file',
         help='WRF geo_em* file name',
     )
-
+    bullet_list = '\n- '
+    parser.add_argument(
+        type=str,
+        dest='wrf_version',
+        help=(
+            f'Expected WRF version numbers:\n- '
+            f'{bullet_list.join(WRF_VERSIONS_DICT.keys())}'
+            f'\nNote: in case you use a version older than v4.3, please specify v4.3\n'
+        ),
+        choices=(WRF_VERSIONS_DICT.keys()),
```

Comment:
we can use `choices` here which automatically takes care of the validation, so we don't have to do this manually.